### PR TITLE
Fix fakezod by tagging proper urbit/arvo version

### PR DESCRIPTION
**Pull request by s-ol**
urbit/arvo#master is not compatible with urbit 0.4.3, making fakezods impossible.

I also only cloned it in fakezod and removed the file you weren't using.

Layer count has been reduced by merging multiple RUN statemets where appropriate.
Urbit and APT are still seperate so the cache can work better when updating only a new urbit version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,17 +34,21 @@ $ docker run -ti -v `pwd`/fintud-macrep:/urbit/fintud-macrep paulbellamy/urbit f
 This can be useful for development
 
 ```
-$ docker run -ti paulbellamy/urbit -F -I zod -A /urbit/arvo -c mycomet
+$ docker run -ti paulbellamy/urbit -F -I zod -A /arvo -c mycomet
 ```
 
-There is another image, with a different entrpoint to make this even
-easier:
+There is another image, with a different entrypoint and the arvo repo checekd
```
Comment (paulbellamy) [inline README.md]: typo *checked

File: README.md [hunk 2]
```diff
@@ -35,17 +34,21 @@ $ docker run -ti -v `pwd`/fintud-macrep:/urbit/fintud-macrep paulbellamy/urbit f
...
+out to make this even easier:
 
-```
-$ docker run -ti paulbellamy/fakeurbit -c mycomet
-```
+    $ docker run -ti paulbellamy/urbit:fakezod -A /arvo -c piername
+
+to start it up after stopping it, omit `-A` and `-c`:
+
+    $ docker run -ti paulbellamy/urbit:fakezod piername
```
Comment (paulbellamy) [inline README.md]: I'm assuming the change from a separate image to a separate tag will mean re-setting up the dockerhub automated builds? It's not a problem, just checking.

**Review by paulbellamy — COMMENTED** [review]
Thanks for the PR! Just a question and typo, then it will be ready to go.

File: README.md (hunk 2 above)
Comment (s-ol) [reply README.md]: yes, you will need to add a second build-line on the original repo in docherhub and remove the other one.

**Comment by s-ol** [comment]
>Better not merge yet, i have been tweaking things a bit and am not sure which state is currently on my master. I'll let you know when I think it is stable and you can do your own testing of course.

I think it's merge-ready now.

**Comment by s-ol** [comment]
Looking better now. I need to test the fakezod and HTTP again, but you will notice i dropped the `useradd` stuff again - docker added a `--user` flag at some point and it was making the signal passing wonky anyway.

I also added an `EXPOSE 8443`, this allows linking the container to something like `nginx-proxy`, which will then autodetect the port with zero configuration.

Btw since you were asking, here is what I would set up the build settings like:
2017-03-05-190958_880x370_scrot

**Comment by paulbellamy** [comment]
Thanks very much!

**Pull request merged**